src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -83,16 +83,12 @@ export default {
 
 		// Add to Basket Interstitial
 		this.initializeAddToBasketInterstitial();
+
 		// Initialize Help Me Choose Experiment
-		if (this.targetedLoanChannel === 'women'
```

Comment:
Looks like `targetedLoanChannel` can be removed from the component.

Reply:
Done